js/gameJudgement.js

```diff
@@ -336,10 +336,12 @@ if("undefined"==typeof jQuery)throw new Error("Bootstrap's JavaScript requires j
+
+var timeInMs=Date.now(),threeMonthsAgo=timeInMs-7776e6,igdb_releases="https://igdbcom-internet-game-database-v1.p.mashape.com/games/";if(igdb_releases+="?filter%5Brelease_dates.date%5D%5Blte%5D="+timeInMs+"&filter%5Brelease_dates.date%5D%5Bgte%5D="+threeMonthsAgo+"&filter%5Brelease_dates.platform%5D%5Beq%5D=6",igdb_releases+="&fields=name%2crating%2caggregated_rating%2crelease_dates%2calternative_names%2ccover.cloudinary_id%2curl&limit=10&offset=0&order=release_dates.date%3Adesc",navigator.userAgent.match(/IEMobile\/10\.0/)){var msViewportStyle=document.createElement("style");msViewportStyle.appendChild(document.createTextNode("@-ms-viewport{width:auto!important}")),document.querySelector("head").appendChild(msViewportStyle)}var app=angular.module("gameJudgement");app.controller("recentReleasesCtrl",["$http","$scope",function($http,$scope){var ctrl=this;ctrl.newReleases={},ctrl.steamList={},ctrl.rightNow=timeInMs,ctrl.threeMonthsAgo=threeMonthsAgo,ctrl.releaseNameCleaner=function(){for(var i=0;i<ctrl.steamList.length;i++)for(var j in ctrl.newReleases)ctrl.newReleases[j].name.toLowerCase()==ctrl.steamList[i].name.toLowerCase()?ctrl.newReleases[j].name=ctrl.steamList[i].name:ctrl.newReleases[j].name.replace(": "," - ").toLowerCase()===ctrl.steamList[i].name.toLowerCase()?ctrl.newReleases[j].name=ctrl.steamList[i].name:ctrl.newReleases[j].name.replace(": "," ").toLowerCase()===ctrl.steamList[i].name.toLowerCase()?ctrl.newReleases[j].name=ctrl.steamList[i].name:ctrl.steamList[i].name.replace("_"," ").toLowerCase()===ctrl.newReleases[j].name.toLowerCase()?ctrl.newReleases[j].name=ctrl.steamList[i].name:ctrl.newReleases[j].name.replace("and","&").toLowerCase()==ctrl.steamList[i].name.toLowerCase()?ctrl.newReleases[j].name=ctrl.steamList[i].name:ctrl.steamList[i].name.replace("®","").toLowerCase()==ctrl.newReleases[j].name.toLowerCase()?ctrl.newReleases[j].name=ctrl.steamList[i].name:ctrl.steamList[i].name.replace("™","").toLowerCase()==ctrl.newReleases[j].name.toLowerCase()?ctrl.newReleases[j].name=ctrl.steamList[i].name:ctrl.steamList[i].name.replace("™","").toLowerCase()==ctrl.newReleases[j].name.replace(": "," ").toLowerCase()?ctrl.newReleases[j].name=ctrl.steamList[i].name:ctrl.steamList[i].name.replace("™","").trim().toLocaleLowerCase()==ctrl.newReleases[j].name.replace(": "," ").toLowerCase()?ctrl.newReleases[j].name=ctrl.steamList[i].name:ctrl.newReleases[j].name.toUpperCase()==ctrl.steamList[i].name&&(ctrl.newReleases[j].name=ctrl.steamList[i].name)},$http.get("/steamList.json").success(function(data,status,headers,config){ctrl.steamList=data,ctrl.releaseNameCleaner()}),ctrl.clickCounter=0,$scope.getReleases=function(){ctrl.clickCounter++,$scope.target||($scope.target=75),$http({method:"GET",headers:{"X-Mashape-Key":"MY_IGDB_API_KEY",Accept:"application/json"},url:igdb_releases}).success(function(data,status,headers,config){ctrl.newReleases=data,ctrl.releaseNameCleaner()})}}]);
```

Comment:
I'm guessing these are the files your site compiles down to. You should put these files in your .gitignore file so you don't check them in. there is no reason to see them in source control